### ISSUE TITLE
iOS only: Downloading a pdf adds a .txt to the filename and renders the file unusable

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#1631]
Connected to a local server here, which might be part of the problem. Maybe the MIME type as reported by the flask server isn't correct.
However, when I try to download a pdf on iOS I get a .txt file and can't use it.

**Comment by urapadmin** [comment]
works in Chrome!

**Comment by urapadmin** [comment]
@arch-kiosk/test:
Chrome has its own download mechanism!!! That is far better than the Apple crap. 
We'll use the Chrome browser from now on! All ipads need it installed.

When downloading a workstation the wiping out the downloadfolder first and all that jazz is unnecessary. Better even: One can send the downloaded file right to filemaker go without tapping nonsensical and superfluous cryptic icons!

**Comment by luizaogs** [comment]
🫢😬😮

**Comment by luizaogs** [comment]
What am I supposed to test this with? I had assumed reports but am getting Excel files for those?

**Comment by urapadmin** [comment]
yes, on PASU at least the finds & whatnot reporting spits out Excel files. But you may focus on the USTP Context sheet report (pdf) or Excel files you download from the file respository (after having file-imported them).

**Comment by lbestock** [comment]
This is where @lbestock is tracking testing pdf downloads from the local PASU server.

- [ ] try on both iOS Chrome and Safari again. 
- [ ] try on Mac Safari
- [ ] try on PC Chrome

**Comment by lbestock** [comment]
Worked absolutely as expected using the local server. I wasn't using a report, though, just a pdf I myself had uploaded.
